R/structural.properties.R

```diff
@@ -2508,6 +2507,14 @@ unfold_tree <- function(graph, mode = c("all", "out", "in", "total"), roots) {
 #' @author Gabor Csardi \email{csardi.gabor@@gmail.com}
 #' @export
 #' @keywords graphs
+#' @section Normalization methods:
+#'
+#' - "unnormalized": Unnormalized Laplacian, \eqn{L = D - A}.
+#' - "symmetric": Symmetric normalized Laplacian,
```

Comment:
```suggestion
#' - "symmetric": Symmetrically normalized Laplacian,
```

(I'll fix this in the C core as well.)